tools/autograd/gen_autograd.py

```diff
@@ -4,6 +4,7 @@
 
 python -m tools.autograd.gen_autograd \
        build/aten/src/ATen/Declarations.yaml \
+       aten/src/ATen/native/native_functions.yaml \
```

Comment:
Why is that? Is this something you added in this PR? Or that was added before and the doc wasn't updated?

Reply:
It was added before the PR but doc wasn't updated :P

Reply:
ok ok
Thanks for fixing that then!